examples/erc721/tests/erc721.rs

```diff
@@ -40,7 +40,13 @@ macro_rules! send {
 
 macro_rules! watch {
     ($e:expr) => {
-        $e.send().await?.watch().await
+        send!($e)?.watch().await
+    };
+}
+
+macro_rules! wait_for_receipt {
```

Comment:
This is not a strong opinion, but what do you think of calling it just `receipt!`?

When people see this macro, they'll immediately jump to the definition, which is a straightforward one-liner. Then, having to type `wait_for_receipt` can become burdensome.